sdk/contentsafety/azure-ai-contentsafety/setup.py

```diff
@@ -13,7 +13,7 @@
 
 
 PACKAGE_NAME = "azure-ai-contentsafety"
-PACKAGE_PPRINT_NAME = "Azure AI Content Safety"
```

Comment:
Nit: we need to keep this how it was before. I'll open an issue for Izzy, maybe we can have a line in the generator config that can set this value

Reply:
Issue: https://github.com/Azure/autorest.python/issues/2207